resources/deployments/controller/controller_deployments_external_test.go

```diff
@@ -44,6 +46,16 @@ func makeDeviceAuthHeader(claim string) string {
 		base64.StdEncoding.EncodeToString([]byte(claim)))
 }
 
+func makeApi(router rest.App) *rest.Api {
+	api := rest.NewApi()
+	api.Use(
+		&requestlog.RequestLogMiddleware{},
```

Comment:
As I understood there's some planned refactoring for transitioning to https://github.com/mendersoftware/go-lib-micro Now that this issue of a custom logger has been raised, it makes sense to add that functionality in the lib (to the benefit of all) and just update deployments when pulling in the lib.